engine/table/src/main/java/io/deephaven/engine/table/impl/select/DhFormulaColumn.java

```diff
@@ -189,7 +189,7 @@ public List<String> initDef(
                     .endl();
 
             applyUsedVariables(columnDefinitionMap, result.getVariablesUsed(), result.getPossibleParams());
-            returnedType = result.getType();
+            returnedType = TypeUtils.getUnboxedTypeIfBoxed(result.getType());
```

Comment:
This seems like a healthy change. That said, should we be changing the consumer of returned type to ensure that things are better for all select columns? Or should we check the other implementations of `getReturnedType`? I'm mainly concerned about the python formula column.